scripts/fiber/record-tests

```diff
@@ -8,8 +8,9 @@ const fs = require('fs');
 const os = require('os');
 const path = require('path');
 
-const SearchSource = require('jest').SearchSource;
-const TestRunner = require('jest').TestRunner;
+const SearchSource = require('jest-cli').SearchSource;
+const TestRunner = require('jest-cli').TestRunner;
+const TestWatcher = require('jest-cli/build/TestWatcher');
```

Comment:
There is a PR open to remove the need to delve into internals. https://github.com/facebook/jest/pull/2138